core/src/i18n/en.yaml

```diff
@@ -780,6 +781,12 @@ domains:
     label: Include Domains
     placeholder: Domain that is allowed
 event-subscription:
+  conditions:
+    last-transition-time: Last transition time
```

Comment:
We have `Last transition` in the jobs view and `Last transition time` here. You can pick any of those, but let's just agree to use the same one in both views.